google/cloud/internal/async_read_write_stream_impl.h

```diff
@@ -144,6 +145,10 @@ class AsyncStreamingReadWriteRpcImpl
     return op->p.get_future();
   }
 
+  RpcMetadata GetRequestMetadata() const override {
+    return google::cloud::internal::GetRequestMetadataFromContext(*context_);
```

Comment:
Q.  Is the full qualification to the current scope necessary/desirable?

Reply:
No. Fixed.